docs/book/src/user/troubleshooting.md

```diff
@@ -116,6 +116,32 @@ Alternatively a Cert Manager yaml file can be placed in the [clusterctl override
 
 More information on the clusterctl config file can be found at [its page in the book](../clusterctl/configuration.md#clusterctl-configuration-file)
 
+## Failed clusterctl upgrade apply - 'failed to update cert-manager component'
+
+Upgrading Cert Manager may fail due to a breaking change introduced in Cert Manager release v1.6.
+An upgrade using `clusterctl` is affected when:
+
+* using `clusterctl` in version `v1.1.4` or a more recent version.
+* Cert Manager lower than version `v1.0.0` did run in the management cluster (which was shipped in Cluster API until including `v0.3.14`).
+
+This will cause `clusterctl upgrade apply` to fail with the error:
+
+```bash
+cluster-api % clusterctl upgrade apply
```

Comment:
```suggestion
clusterctl upgrade apply
```
nit. I think (xref: https://github.com/kubernetes-sigs/cluster-api/pull/6521/files)

Reply:
Its the same as on that PR (or at the paragraph above)

Reply:
I think the prefix is actually only above in this doc, right? As far as I can see the `cluster-api %` is not used anywhere else